packages/renderer/src/PreferencesNavigation.svelte

```diff
@@ -186,7 +205,10 @@ $: addSectionHiddenClass = (section: string): string => (sectionExpanded[section
                   href="/preferences/default/{configItem.id}"
                   id="configuration-section-{configSection.toLowerCase()}-{configItem.title.toLowerCase()}"
                   class="pf-c-nav__link"
-                  style="font-weight: 200">{configItem.title}</a>
+                  style="font-weight: 200"
+                  aria-label="{configItem}"
```

Comment:
I think it is missing the attribute on the configItem probably id or title

Reply:
nice catch.